### PR TITLE
Components and styles for the aliados section

### PR DESCRIPTION
**Pull request by dyepes62**
**Description**
Components and styles for the aliados section, I am waiting for the correct brands in order to see the styles on those.
Please remember to do **npm-install** after including these changes.
*It was necessary for me to use an important (css) to be able to modify the bullets of the carousel and not lose the work done with swiper.

**Ticket trello**
https://trello.com/c/sbAjelgK/45-home-aliados-funcionalidad-y-dise%C3%B1o-sp5

**Screenshots**
image

image

### CODE REVIEW

**Comment by coding-tomato** [comment]
Hello! I'm struggling to get the branch to display properly. Any idea on why I can't replicate your environment?

I pulled your branch, did **npm install** then **npm run dev**

Mobile view
imagen
Desktop view
imagen

The whole component looks huge, not quite sure about the reason.

Edit (still failing):
I also attempted to re-clone the repo into another directory, still the same result.

Edit (found the problem):
It seems to display this error on Firefox, still unsure about the reason.

File: components/AliadosSlide.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+        <img
+          className="aliadosSlide__image"
+          alt={alternativeText}
+          src={image}
+        />
+      </section>
+      <section className="aliadosSlide__brands">
+        {brandLogos.map((logo, index) => (
+          <img
```
Comment (SinuheM) [inline components/AliadosSlide.js:23]: This map without a key are going to show this warning 
`Each child in a list should have a unique "key" prop`

You could use for example the title field and put something like
`<img key="logo.fields.title" ...`

File: components/AliadosCarousel.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          clickable: true,
+          type: 'bullets',
+          bulletElement: 'span',
+          bulletClass: 'aliadosBullet',
+          bulletActiveClass: 'aliadosBullet-active',
+        }}
+        className="aliadosCarousel"
+      >
+        {carouselComponents.map((slide) => (
+          <SwiperSlide>
```
Comment (SinuheM) [inline components/AliadosCarousel.js]: This map without a key are going to show this warning
`Each child in a list should have a unique "key" prop`

You could look for an identifier and put something like
`<SwiperSlide key="identifier" ...`

**Review by SinuheM — APPROVED** [review]

**Review by coding-tomato — APPROVED** [review]
Looks good!

**Pull request merged**